### DISCUSSION TITLE
Introduce `Java17` label

### DISCUSSION THREAD

**Discussion by zakkak** [oracle/graal#4246] [Ideas] [answered]
Hello all,

I suggest adding a 'Java17' label to github to mark issues related to java 17 support, e.g.:

* https://github.com/oracle/graal/issues/4006
* https://github.com/oracle/graal/issues/3984
* etc.

WDYT?

**Comment by alina-yur** [comment, marked as answer]
Hi, makes sense to me. @dougxc @gilles-duboscq what do you think?

**Reply by gilles-duboscq** [reply]
Yes, i agree, we already have `Java8` we should add `Java11` and `Java17`.

**Reply by alina-yur** [reply]
Added both. Thank you for suggesting, @zakkak :)

**Reply by zakkak** [reply]
Thank you @alina-yur
